src/QmlControls/QGroundControlQmlGlobal.cc

```diff
@@ -41,11 +44,30 @@ QGroundControlQmlGlobal::QGroundControlQmlGlobal(QGCApplication* app, QGCToolbox
 {
     // We clear the parent on this object since we run into shutdown problems caused by hybrid qml app. Instead we let it leak on shutdown.
     setParent(NULL);
+    // Load last coordinates and zoom from config file
+    QSettings settings;
+    
+    settings.beginGroup(_flightMapPositionSettingsGroup);
+    _coord.setLatitude(settings.value(_flightMapPositionLatitudeSettingsKey, 0).toDouble());
+    _coord.setLongitude(settings.value(_flightMapPositionLongitudeSettingsKey, 0).toDouble());
+
+    settings.beginGroup(_flightMapPositionSettingsGroup);
```

Comment:
I get it this came to be because of a copy and paste but you might as well remove the redundant `settings.beginGroup()` for the zoom level.